src/Router.php

```diff
@@ -17,14 +17,71 @@ public static function getInstance()
         return static::$instance;
     }
     
+    /**
+     * Loads the location of the routes file.
+     * 
+     * @return string
+     */
+    protected function getRouteFileLocation()
+    {
+        $dir = getenv('HOME') . '/.volcano-sdk';
+        
+        if (!is_dir($dir)) {
+            mkdir($dir);
+        }
+        
+        return $dir . '/routes.json';
+    }
+    
+    /**
+     * Loads Routes
+     */
     public function loadRoutes()
     {
+        $rfile = $this->getRouteFileLocation();
+        
+        if (!file_exists($rfile) || time() - filemtime($rfile) > 86000) {
```

Comment:
So this will load from service every ~24 hours?

Nice file modified timestamp check - no need to update the Endpoints API response! :+1: